content/docs/networking/load-balancing.md

```diff
@@ -1,47 +1,99 @@
+This abstraction is enabled by the **Service IPs**. These are a set of addresses that identify services and all their instances and that can be used to transparently pick the right instance and establish a connection. A Service IP expresses an inherent balancing policy, and, for each service, we have as many Service IPs as the implemented system balancing policies.
 
 A subset of the Service IPs are the **Instance IPs**. They balance the traffic always to a specific instance of a service. Therefore, when deploying a service, the system will provision the following addresses:
 
 * 1 Load Balanced Service IP for each balancing policy implemented in the system
 * 1 Instance IP for each new instance of the service that has been deployed. 
 
-The latest version of Oakestra (v0.4.201), only implements Round Robin Service IPs. 
+The latest version of Oakestra (v0.4.301), only implements Round Robin Service IPs. 
+
+<details>
+<summary><b>What's the difference between an Instance IP and a Namespace IP?</b></summary>
+They operate on 2 different abstraction layers. The Namespace IP depends on the virtualized subnetwork of a worker node and changes when migrating an application from one node to another. It cannot be provisioned beforehand and must not be used by developers. The Instance IP is part of the Service layer abstraction. Therefore it identifies an instance regardless of migration operations, scales up, scales down, and can be provisioned even before the deployment of a service instance.  
+</details>
+
+### Semantic Addressing
+
+The semantic addressing happens on the above mentioned service layer.
+Similar to a cluster IP in Kubernetes, these addresses reference all the instances (replicas) of a microservice with a single address. This address does not change when scaling up the instances or when migrating them.
+Anyway, unlike the Kubernetes cluster IP, when deploying a service in Oakestra, the platform provides as many Service IP addresses as the number of balancing policies supported (and active) within the platform.
+
+#### Example
+
+[NetExample](/network/NetExample_dual_stack.png)
+
+A developer can communicate with any instance of Service B either with **Round Robin** balancing policy or the **Closest** balancing policy. The former balances the traffic evenly between all the instances, and the latter finds the geographically closer instance.
+
+**Reminder**: Currently the **IPv6 Closest** balancing strategy IP addresses (reserved, but not implemented) are in the `fdff:1000::/21` subnet, and **IPv6 Round Robin** in `fdff:2000::/21`. In IPv4, both balancing strategies take their IPs from the `10.30.0.0/16` subnet. For further information, take a look at the [IPv4](/networking/ipv4-address-space.md) or [IPv6](/networking/ipv6-address-space.md) address space documentation pages.
```

Comment:
In-line comment

Reply:
everything describing the messaging requests?